libtenzir/include/tenzir/curl.hpp

```diff
@@ -345,6 +349,9 @@ class easy {
   /// `curl_easy_reset`
   auto reset() -> void;
 
+  /// Checks that the
+  auto validate_cacert(diagnostic_handler& dh) -> bool;
```

Comment:
The idea of the curl abstraction is to keep it *un*opinionated, i.e., do not creep Tenzir logic into it.

Can we use free functions for that?